catalog/spatial/registry/registry-rest-endpoint/src/main/resources/templates/organizations.hbt

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html; charset=windows-1252" http-equiv="content-type"/>
```

Comment:
I would expect the charset to be UTF-8.  Is there a reason it is `windows-1252`?